app/src/main/java/com/dessalines/thumbkey/keyboards/ESMessageEase.kt

```diff
@@ -640,13 +692,50 @@ val KB_ES_MESSAGEEASE_SHIFTED =
         ),
     )
 
+val NUM_KEY_2 = NUMERIC_KEYBOARD_MESSAGEASE.arr[0][1]
+val NUM_KEY_3 = NUMERIC_KEYBOARD_MESSAGEASE.arr[0][2]
+val KB_ES_MESSAGEEASE_NUM =
+    NUMERIC_KEYBOARD_MESSAGEASE.copy(
+        listOf(
+            listOf(
+                NUMERIC_KEYBOARD_MESSAGEASE.arr[0][0],
+                NUM_KEY_2.copy(
+                    center = NUM_KEY_2.center,
+                    swipes =
+                        NUM_KEY_2.swipes?.plus(
+                            SwipeDirection.RIGHT to
+                                KeyC(
+                                    display = KeyDisplay.TextDisplay("¡"),
+                                    action = KeyAction.CommitText("¡"),
+                                ),
```

Comment:
this is still wrong

Reply:
This is intended behavior. The closing symbols are in the alphabetic keyboard and the opening ones are in the numeric keyboard (the best approximation to the usability of MessagEase's ES layout I could find)

Reply:
so sorry, I thought I was looking at the english numeric, my bad!